server/src/database.js

```diff
@@ -99,7 +99,7 @@ export async function listContainersByUser(values = []) {
 
 export async function removeContainer({
   id,
-}) { return db.run('DELETE FROM containers WHERE id=?', [id]) }
+}) { return db.run('DELETE FROM containers WHERE subdomain=?', [id]) }
```

Comment:
In the feature-threads branch I have added a db.del(...) Command which returns number of rows affected. Should probably be used here later to detect if delete was successful.